### ISSUE TITLE
Can't create a new ressource group after upgrading to 1.12 with postgres

### ISSUE THREAD

**Issue by fondemen** [LINBIT/linstor-server#231]
Hello,
I have an issue after updating linstor using Postgres 12.6 as a backend when trying to create a new ressource group : https://gist.github.com/fondemen/a69c719a42274c9acb340ab4a76cc990

I don't know if it's a schema issue ; logs show:
```
11:35:42.640 [grizzly-http-server-0] ERROR LINSTOR/Controller - SYSTEM - Could not set object '[]' of type String as SQL type: 2005 (CLOB) for column RESOURCE_GROUPS.NODE_NAME_LIST [Report number 609282FD-00000-000001]
11:35:42.717 [grizzly-http-server-1] WARN  LINSTOR/Controller - SYSTEM - Path '/v1/resource-definitions//resources' not found on server.
```
and 
```
Cannot cast an instance of java.lang.String to type Types.CLOB
```

Interestingly, it's the only issue I've experienced so far: i can create resources, volumes, and see existing resource groups...

(issue also reported in  kvaps/kube-linstor#39)

Regards,

**Comment by fondemen** [comment]
Just to be more precise, the resource_groups::node_name_list column was changed from `character varying(4096) DEFAULT '[]'::character varying` to `text DEFAULT '[]'::character varying`.

**Comment by fondemen** [comment]
Reproduced migration from 1.11.1 to 1.12.2:
 ```

20:32:07.518 [Main] INFO  org.flywaydb.core.internal.license.VersionPrinter - Flyway Community Edition 6.5.7 by Redgate
20:32:07.591 [Main] INFO  org.flywaydb.core.internal.database.DatabaseFactory - Database: jdbc:postgresql://linstor-db/linstor (PostgreSQL 10.6)
20:32:07.636 [Main] INFO  org.flywaydb.core.internal.command.DbValidate - Successfully validated 56 migrations (execution time 00:00.027s)
20:32:07.645 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Current version of schema "LINSTOR": 2020.12.03.10.00
20:32:07.645 [Main] WARN  org.flywaydb.core.internal.command.DbMigrate - outOfOrder mode is active. Migration of schema "LINSTOR" may not be reproducible.
20:32:07.650 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2020.10.13.12.40 - Add EXTERNAL_LOCKING column with default value false [out of order]
20:32:07.700 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2020.10.22.13.42 - Add INACTIVE_PERMANENTLY flag to already INACTIVE DRBD resources [out of order]
20:32:07.719 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2021.02.24.12.00 - Add allowed verify algorithm list
20:32:07.753 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2021.03.30.12.00 - Add StorPoolDiskless column to autoplacer config in ResourceGroup
20:32:07.836 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2021.04.12.12.00 - Add Files table
20:32:07.906 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2021.04.29.12.00 - Fix duplicated restored layer data
20:32:07.922 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2021.04.29.13.00 - Fix incorrectly written data after migration 2021.03.30.12.00
20:32:07.936 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Migrating schema "LINSTOR" to version 2021.05.03.13.00 - Fix incorrectly written data after migration 2021.03.30.12.00 and 2021.04.29.*
20:32:08.029 [Main] INFO  org.flywaydb.core.internal.command.DbMigrate - Successfully applied 8 migrations to schema "LINSTOR" (execution time 00:00.388s)
```

**Comment by rp-** [comment]
Thanks for the report, should be fixed in v1.12.3

**Comment by fondemen** [comment]
Problem solved !
Thanks a lot !
